### ISSUE TITLE
Create a new .php that will serve as our main landing page URL

### ISSUE THREAD

**Issue by jdhtwo** [mattcroce/HSver1#7]
We can later link the hc.gov and savisky model to pages beyond this, but having the css set up now will save us time later, and it's something I can do now.

**Comment by mattcroce** [comment]
Yeah - go for it! I'm excited to see what you come up with!
Only question - why make it a .php page if it's gonna function as a landing page? Are you foreseeing a need for writing some php into this page? If so, that's cool. If not, could probably be .html - if that matters.

**Comment by jdhtwo** [comment]
Good point. I think I was stuck in our .php vs. .html conversations. It can
certainly be an .html page.

Going to start modeling off of Carousel [
http://getbootstrap.com/examples/carousel/]

**Comment by jdhtwo** [comment]
homepage.html created and uploaded to Github as well as bluehost.

Next step is to link this to become our main followthesage.com / healthsage.info URL.

**Comment by jdhtwo** [comment]
@mattcroce  - what else needs to happen before we are ready to call homepage.html our actual homepage?

**Comment by mattcroce** [comment]
@jdhtwo 
I guess not too much, but a couple things definitely still need to happen:
1. We'd want to use followthesage.com, rather than /homepage.html, right? So the stuff that is at followthesage.com right now would need to be cleared out and replaced.
2. Current followthesage.com/homepage.html needs to be deleted as it was used an old tester for me.
3. I think that right now followthesage.com/homepage/homepage.html still says "Matt Croce Is My Hero". And while I appreciate the sentiment and it really does mean a lot to me, we should probably not have that as our homepage. Additionally, that homepage.html needs to be fleshed out with some more material, I think.

So, to summarize:
Use followthesage.com as our homepage. That means transition the current css/template from followthesage.com/homepage/homepage.html to followthesage.com and also we need to finish putting the basic homepage info into the new followthesage.com. Does that make sense?

I'm heading home now and will plan to check in online tonight. Feel free to text or email questions! Hope that wasn't confusing!

**Comment by jdhtwo** [comment]
**1. We'd want to use followthesage.com, rather than /homepage.html, right? So the stuff that is at followthesage.com right now would need to be cleared out and replaced.**
I concur. Pretty easy step. I am happy to do this. I feel strongly that in the end we should still use healthsage.info as opposed to followthesage.com. If jumping to followthesage.com is easier now for URL pointing purposes, then let's do that.
**2. Current followthesage.com/homepage.html needs to be deleted as it was used an old tester for me.**
I concur, but I think you mean just followthesage.com, and not followthesage.com/homepage.html, right? Pretty easy step. I am happy to do this.
**3. I think that right now followthesage.com/homepage/homepage.html still says "Matt Croce Is My Hero". And while I appreciate the sentiment and it really does mean a lot to me, we should probably not have that as our homepage. Additionally, that homepage.html needs to be fleshed out with some more material, I think.**
I concur. See #21.

Would it make sense to merge "master" and "homepage" in Github, or is it easier to keep these separate?

**Comment by mattcroce** [comment]
1. We'd want to use followthesage.com, rather than /homepage.html, right? So the stuff that is at followthesage.com right now would need to be cleared out and replaced.

**I concur. Pretty easy step. I am happy to do this. I feel strongly that in the end we should still use healthsage.info as opposed to followthesage.com. If jumping to followthesage.com is easier now for URL pointing purposes, then let's do that.**

**_I agree we want to ultimately use healthsage.info as our homepage, but I think until we are done building out something that we are really happy with at followthesage.com, we should use that as our sort of development environment before we migrate to healthsage.info. I think followthesage.com still holds value and could be a place where folks are pointed for other answers or where the actual tool lives. Something along the lines of healthsage.info being the information repository, FAQ, introduction to HealthSage, etc. and when someone is interested and wants to use the tool or sign up, they click a "follow the sage" button, or whatever and it shoots them to followthesage.com where the tool actually lives. I don't know - just kinda spitballing here.**_
1. Current followthesage.com/homepage.html needs to be deleted as it was used an old tester for me.

**I concur, but I think you mean just followthesage.com, and not followthesage.com/homepage.html, right? Pretty easy step. I am happy to do this.**

**_No, I did mean followthesage.com/homepage.html. That is something that exists right now that needs to go away. We do definitely want to use followthesage.com as our homepage. Essentially, there will be a file hosted at bluehost called index.html or default.html within the main folder for followthesage and that file is the homepage file. When someone visits followthesage.com, it will load the index.html file but without having to type "/index.html" after followthesage.com. It has something to do with file structures and how the directories are read and I don't totally understand it right now, but that's basically how it works.**_
1. I think that right now followthesage.com/homepage/homepage.html still says "Matt Croce Is My Hero". And while I appreciate the sentiment and it really does mean a lot to me, we should probably not have that as our homepage. Additionally, that homepage.html needs to be fleshed out with some more material, I think.

**I concur. See #21.**

**_Awesomesauce.**_

Would it make sense to merge "master" and "homepage" in Github, or is it easier to keep these separate?

**_I'm not sure, really. I think my gut feeling is to keep them separate until we have something more workable completed for the homepage and then clean up what we have in master - I think we'll lose a lot of what's there (if I'm remembering what's there correctly, I didn't go back to check) and will have just our css and our index.html(homepage) in there and then we can work on the other pages?**_

**Comment by jdhtwo** [comment]
Closed #21, which addresses Point 3 here in this thread about getting the site more "professional".

**Comment by jdhtwo** [comment]
Ok - going to follow these steps above:
1. Delete current followthesage.com/homepage.html
2. Upload my latest homepage.html and associated CSS/Pics folders to Bluehost. Use my latest homepage.html as index.html.

**Comment by mattcroce** [comment]
That sounds good, you'll just need to be careful of the folder structure that you create so that the files are all located properly.

The current followthesage.com is already called index.html, so you'll just need to overwrite the contents of the current index.html with the contents of your homepage.html.

**Comment by jdhtwo** [comment]
Cool. Done.

Yep - that was exactly my plan - I simply did an overwrite of index.html with my current homepage.html. I am going to change my local homepage.html name to index.html.

I added css and pics folder because that's how my local file structure was arranged.

For our future reference, I'm using files from this homepage branch commit: https://github.com/mattcroce/HSver1/commit/c25df91c9fd29a0699f617d889581ad12e534956 with one change that Matt caught that will go in the next commit - there was a return in the "start comparing" html link to the savisky model. I fixed it manually in Bluehost as well as in my local file.

I saved your former index.html locally in case you wanted it and didn't have a backup. I left all other files as is. I did not see a homepage.html file of yours - I just saw my homepage folder, which contains a homepage.html file.

Closing this issue because we've addressed the main topic! All of the other items can be reopened as new items.
